.coveragerc

```diff
@@ -0,0 +1,36 @@
+[run]
+branch = True
+include=
+    # Include the following source code folders
+    **/lambdas/**
+    **/tests/**
```

Comment:
We need to exclude tests folders. Otherwise the tests themselves count as covered which dilutes the overall test coverage which results in an incorrect coverage number.